platform/event/event_core.proto

```diff
@@ -0,0 +1,80 @@
+/**
+ * # Core Event Data
+ * A message that describes the metadata for an event.
+ *
+ * The `EventCore` contains a list of the event's parents, as well as the software
+ * version, an identifier for the node that created this event, the birth round, and
+ * the creation timestamp for the event.
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.platform.event;
+
+/*
+ * Hedera Network Services Protobuf
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import "event/event_descriptor.proto";
+import "timestamp.proto";
+import "basic_types.proto";
+
+option java_package = "com.hedera.hapi.platform.event";
+// <<<pbj.java_package = "com.hedera.hapi.platform.event">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * Contains information about an event and its parents.
+ */
+message EventCore {
+  /**
+   * The software version of the node at the time the event was created.<br/>
+   * This software version SHALL exactly match the software version passed into the platform at construction.
+   */
+  proto.SemanticVersion version = 1;
+
+  /**
+   * A Creator Identifier.<br/>
+   * This SHALL be the unique identifier for the node that created the event.
+   */
```

Comment:
Is it worth also documenting this ID is the same as the node ID in the address book for this node? Or is it something different? creator_node_id maybe?

Reply:
It is the same. I modified the documentation, and the name of the field, to reflect this